_posts/2022-11-20-nhibernate-5-4-released.md

```diff
@@ -1,7 +1,7 @@
 ---
 layout: post
 title: "NHibernate 5.4 Released"
-date: 2020-11-20T17:20:00Z
+date: 2022-11-20T17:20:00Z
```

Comment:
The year from previous release was not adequately updated in the 5.4 announcement.